### PR TITLE
Feature/356 get pre defined assets from blob storage

### PR DESCRIPTION
**Pull request by rodrigo-baldiviezo-jalafund**
[AB#356](https://dev.azure.com/funjaladev/DevLevel3/_boards/board/t/DevLevel3%20Team/Stories/?workitem=356)

# Overview

Get pre-defined assets from blob storage

### CODE REVIEW

File: src/constants/images.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+const CONTAINER = "images";
+const DIR = "webapp";
+const BASE_URL = `${process.env.REACT_APP_STORAGE_URL}/${CONTAINER}/${DIR}`;
+
+export const logoHome = `${BASE_URL}/logo-teamup2.594bdda4.png`;
+export const logoNavbar = `${BASE_URL}/logo-teamup1.png`;
+export const notFound = `${BASE_URL}/not-found.jpg`;
+export const loadingGift = `${BASE_URL}/loading.gif`;
+export const avatar = `${BASE_URL}/img_avatar.jpg`;
+export const image = `${BASE_URL}/project-img.jpeg`;
```
Comment (jorge-lema-jalafund) [inline src/constants/images.js]: better naming please

File: src/constants/images.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+export const projectPhoto = `${BASE_URL}/project-img.svg`;
+export const user = `${BASE_URL}/user-img.svg`;
+export const User = `${BASE_URL}/user.svg`;
```
Comment (jorge-lema-jalafund) [inline src/constants/images.js]: better naming please

**Review by jorge-lema-jalafund — CHANGES_REQUESTED** [review]

File: src/constants/images.js (hunk 1 above)
Comment (rodrigo-baldiviezo-jalafund) [reply src/constants/images.js]: done

File: src/constants/images.js (hunk 2 above)
Comment (rodrigo-baldiviezo-jalafund) [reply src/constants/images.js]: done

**Review by jorge-lema-jalafund — APPROVED** [review]

**Review by paulo-teran-jalafund — APPROVED** [review]

**Review by lina-ramirez-jalafund — APPROVED** [review]

**Pull request merged**